sdk/datafactory/tests.mgmt.yml

```diff
@@ -10,3 +10,8 @@ extends:
       - name: azure-resourcemanager-datafactory
         groupId: com.azure.resourcemanager
         safeName: azureresourcemanagerdatafactory
+    Clouds: 'Public'
+    UseFederatedAuth: true
```

Comment:
I remember Wes asked us not to use `UseFederatedAuth`

Reply:
Thanks, I'll remove these two lines. 
Major change here is fixing storage account name conflict. Seems that storage names are global, and all sa00000-sa99999 have been occupied..
Also limiting the test to run only on one agent.